bytes/Makefile

```diff
@@ -60,7 +60,7 @@ migrate: ## Run alembic migrations
 ##|------------------------------------------------------------------------|
 
 test: utest itest ## Run all tests.
-ci-docker-compose := docker compose -f base.yml  -f .ci/docker-compose.yml
```

Comment:
By changing the paths in the compose file you can sometimes resolve this already even without the --project-directory 👍

Reply:
I am not sure what you mean @Donnype, can you be more specific?
* You want to change the path inside the compose file (tried that with `context` wasn't trivial to get to work...)?
* You want to change the path of the compose file?
```
--project-directory string   Specify an alternate working directory
                             (default: the path of the, first specified, Compose file)
```